src/vm/dispatchinfo.cpp

```diff
@@ -749,11 +750,7 @@ void DispatchMemberInfo::SetUpParamMarshalerInfo()
         }
         else if (MscorlibBinder::IsClass(pMemberInfoMT, CLASS__FIELD))
         {
-            MethodDescCallSite getFieldHandle(METHOD__RTFIELD__GET_FIELDHANDLE, &MemberInfoObj);
-            ARG_SLOT arg = ObjToArgSlot(MemberInfoObj);
-            FieldDesc* pFld = (FieldDesc*) getFieldHandle.Call_RetLPVOID(&arg);
-            if (pFld)
-                SetUpFieldMarshalerInfo(pFld);
+            // We don't support non-default marshalling behavior for field getter/setter stubs invoked via IDispatch.
```

Comment:
Perhaps assert this case or is this supposed to be a nop?

Reply:
This has been a no-op with a TODO since moving to GitHub.